README.md

```diff
@@ -82,13 +93,13 @@ Hello World
 _myEntryPoint.ts:_
 ```typescript
 
-console.log(Leaf.readTextFileSync("./resources/hello.txt"));
+console.log(Deno.readTextFileSync("./resources/hello.txt"));
 ```
 
-```batch
-deno run --allow-all --unstable _myEntryPoint.ts
+```shell
+deno run --allow-all --unstable compile.ts
```

Comment:
Maybe we should note the implications of `--allow-all` but that is worth another issue.